manifest.json

```diff
@@ -280,6 +280,27 @@
           "js": ["select2.js"]
         }
       },
+      "ckeditor": {
+        "version": "4.5.8",
+        "downloadFormat": "zip",
+        "url": "http://download.cksource.com/CKEditor/CKEditor/CKEditor%204.5.8/ckeditor_4.5.8_standard.zip",
+        "rootDir": "ckeditor",
+        "targetDir": "ckeditor"
+      },
+      "widget": {
+        "version": "4.5.8",
+        "downloadFormat": "zip",
+        "url": "http://download.ckeditor.com/widget/releases/widget_4.5.8.zip",
+        "rootDir": "widget",
+        "targetDir": "widget"
```

Comment:
Can this be ckeditor-widget, ckeditor-lineutils? I think it's good to tie this back to ckeditor in order to avoid confusion.